carto-package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "postgis",
+    "current_version": {
+        "requires": {
+            "postgres": ">=10.1.1"
+        },
+        "works_with": {
+            "builder": ">=4.12.74"
```

Comment:
Out of curiosity: why 4.12.74?

Reply:
It's just the latest builder release published. And I just fixed something wrong in the code:

https://github.com/CartoDB/cartodb/pull/14235

that should've been updated when we migrated to PG 10, but we forgot about it.

With that I just want to state that the only component that actually depends on `odbc_fdw` is builder. And I'd change and restrict it to the latest version once that PR goes ahead.

Reply:
"restrict" / recommend

Reply:
https://github.com/CartoDB/postgis/pull/26/files#r209941440 this comment apies here too, in this case we don't need to specify on odbc_fdw the "works_with" for builder.

Reply:
In the end I can put here whatever I want, as long as you don't have a tool, builds and tests to check it XD

Reply:
Well, in this case, odbc_fdw specifying builder will affect the version decision.